src/main/java/com/spider/solomon/order/OrderItemDao.java

```diff
@@ -0,0 +1,31 @@
+package com.spider.solomon.order;
+
+import java.util.List;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import jakarta.annotation.Resource;
+
+@Repository
+public class OrderItemDao {
+	@Resource
+	JdbcTemplate jdbcTemplate;
```

Comment:
Not a code-review comment, but I would ask about field injection vs constructor injection. I've read that constructor injection is generally preferred due to testability and surfacing issues of a class with too many dependencies